pkg/apis/cosigned/v1alpha1/clusterimagepolicy_types.go

```diff
@@ -115,6 +115,8 @@ type KeyRef struct {
 type Source struct {
 	// +optional
 	OCI string `json:"oci,omitempty"`
+	// +optional
```

Comment:
Let's add a description here `// SignaturePullSecrets is an optional list of references to secrets in the same namespace to use for pulling any of the signatures used by this Source.`